system/lpc176x/pinmapping.h

```diff
@@ -97,7 +110,91 @@ using LPC176x::pin_t;
 #define P2_11 0x4B
 #define P2_12 0x4C
 #define P2_13 0x4D
+#define P2_14 0x4E
+#define P2_15 0x4F
+#define P2_16 0x50
+#define P2_17 0x51
+#define P2_18 0x52
+#define P2_19 0x53
+#define P2_20 0x54
+#define P2_21 0x55
+#define P2_22 0x56
+#define P2_23 0x57
+#define P2_24 0x58
+#define P2_25 0x59
+#define P2_26 0x5A
+#define P2_27 0x5B
+#define P2_28 0x5C
+#define P2_29 0x5D
+#define P2_30 0x5E
+#define P2_31 0x5F
+
+#define P3_00 0x60
+#define P3_01 0x61
+#define P3_02 0x62
+#define P3_03 0x63
+#define P3_04 0x64
+#define P3_05 0x65
+#define P3_06 0x66
+#define P3_07 0x67
+#define P3_08 0x68
+#define P3_09 0x69
+#define P3_10 0x6A
+#define P3_11 0x6B
+#define P3_12 0x6C
+#define P3_13 0x6D
+#define P3_14 0x6E
+#define P3_15 0x6F
+#define P3_16 0x70
+#define P3_17 0x71
+#define P3_18 0x72
+#define P3_19 0x73
+#define P3_20 0x74
+#define P3_21 0x75
+#define P3_22 0x76
+#define P3_23 0x77
+#define P3_24 0x78
 #define P3_25 0x79
 #define P3_26 0x7A
+#define P3_27 0x7B
+#define P3_30 0x7E
```

Comment:
Any particular reason to leave out P3_28 and P3_29?

Reply:
Probably not particularly relevant, so shoved em in!